src/fixtures/userFakeTestData.js

```diff
@@ -0,0 +1,11 @@
+import faker from 'faker';
```

Comment:
For fixture files, use the format `<model name in lowercase>.fixture.js`. E.g. `user.fixture.js`